docs/source/pages/using.rst

```diff
@@ -251,6 +267,32 @@ Then the analysis can be run with:
 
 With ``path_to_sbom`` is the path to the SBOM you want to use.
 
+-------------------------------------------------------
+Analyzing dependencies using Python virtual environment
+-------------------------------------------------------
+
+Macaron can automatically identify and analyze the dependencies of a Python package if you provide the path to the virtual environment where the package is installed.
+
+Let's say you want to analyze ``django@5.0.6`` and its dependencies. First create a virtual environment and install ``django@5.0.6``:
+
+.. code-block:: shell
+
+  python3.11 -m venv /tmp/.django_venv
```

Comment:
Does using `python3.11` here indicate that for this particular use case, Python3.11 must be used?